src/frontend/Komponenter/Behandling/Formkrav/Formkrav.tsx

```diff
@@ -30,16 +34,22 @@ export const Formkrav: React.FC<{ behandling: Behandling }> = ({ behandling }) =
         }
     }, [behandling, fagsystemVedtak, hentFagsystemVedtak]);
 
+    useEffect(() => {
+        if (klagebehandlingsResultater.status === RessursStatus.IKKE_HENTET) {
+            hentKlagebehandlingsResultater(behandling);
+        }
+    }, [behandling, klagebehandlingsResultater, hentKlagebehandlingsResultater]);
```

Comment:
Nå får vi tre useEffects som fyrer av fra Formkrav fanen for å hente hver sine data. Siden disse tre useEffectene er koblet til metoder som kun gjøres i Formkrav-komponenten kunne man ha vurdert å slå sammen alle disse tre metodene til en metode som henter all dataen som Formkrav-komponenten har behov. Slik kunne man ha sluppet unna med å gjøre kun et kall i stedet for tre. 

Vet ikke helt om det er verdt å skrive om slik det er nå, dette vil også fungere. Men det kan være verdt å tenke over til senere. Hvis man i fremtiden for eksempel ser at man må legge til en fjerde useEffect som kun skal brukes her så ville jeg skrevet om :)